R/MicrobiomeBenchmarkData.R

```diff
@@ -239,6 +239,6 @@ removeCache <- function(ask = interactive()) {
 #'
 #' @keywords internal
 .getSampleMetadata <- function() {
-    data('sampleMetadata', package = 'MicrobiomeBenchmarkData')
+    utils::data('sampleMetadata', package = 'MicrobiomeBenchmarkData')
     sampleMetadata
```

Comment:
see this section on how to use exported data within the package and avoid putting it in the `.GlobalEnv` 
http://contributions.bioconductor.org/data.html?q=adding#adding-data-to-existing-package

Reply:
Thanks, @LiNk-NY, I created a function with some code as in the example: [e8d7862](https://github.com/waldronlab/MicrobiomeBenchmarkData/pull/8/commits/e8d78622bd1570dbcdeaf89b14e3d30446b1cba3)